src/components/Pagination/BreadcrumbList.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import StyledBreadcrumbList from "./BreadcrumbList.styles";
+
+const BreadcrumbList = ({ delimiter, children, ...props }) => {
+  const childrenLen = children.length;
+  return (
+    <StyledBreadcrumbList
+      itemScope
+      itemType="http://schema.org/BreadcrumbList"
+      {...props}
+    >
+      {children.reduce((memo, child, i) => {
+        memo.push(child);
+
+        if (i !== childrenLen - 1) {
+          memo.push(delimiter);
+        }
+
+        return memo;
+      }, [])}
+    </StyledBreadcrumbList>
+  );
+};
+
+BreadcrumbList.propTypes = {
+  children: PropTypes.node.isRequired,
+  delimiter: PropTypes.node.isRequired
```

Comment:
Good point

Reply:
Unfortunately, it doesn't seem like `styled-components` supports passing props into nested selectors, despite the documentation saying otherwise 🤔

Works:
```
  &:not(:last-of-type):after {
    content: '/';
  }
```

```
Doesn't work:
${({ delimiter }) =>
    delimiter &&
    `
    &:not(:last-of-type):after
    content: ${delimiter};
  }
  `
```